app/templates/orders/view.hbs

```diff
@@ -24,12 +24,12 @@
     </div>
   </div>
   <div class="row">
-    <div class="column right aligned">
-      <button class="ui labeled icon {{if isLoadingTickets 'loading'}} button" {{action downloadTickets model.event.name model.order.identifier}}>
+    <div class="column right {{if device.isMobile 'center'}} aligned">
+      <button class="ui labeled icon {{if isLoadingTickets 'loading '}}button" {{action downloadTickets model.event.name model.order.identifier}}>
         <i class="ticket alternate icon"></i>
         {{t 'Download Tickets'}}
       </button>
-      <button {{action 'downloadInvoice' model.event.name model.order.identifier }} class="ui labeled icon blue {{if isLoadingInvoice 'loading'}} button">
+      <button {{action 'downloadInvoice' model.event.name model.order.identifier }} class="ui labeled icon blue {{if isLoadingInvoice 'loading'}}button">
```

Comment:
Unwanted change

Reply:
Should I remove these unwanted changes or not?

Reply:
What is the benefit of keeping the unwanted change? Also follow semantic PR naming